client/src/containers/NychaPage.js

```diff
@@ -313,12 +314,11 @@ class NychaPageWithoutI18n extends Component {
                   url={
                     usersInputAddress &&
                     encodeURI(
-                      "https://whoownswhat.justfix.nyc/address/" +
-                        usersInputAddress.boro +
-                        "/" +
-                        usersInputAddress.housenumber +
-                        "/" +
-                        usersInputAddress.streetname
+                      `https://whoownswhat.justfix.nyc${createRouteForAddressPage(
+                        usersInputAddress.boro,
+                        usersInputAddress.streetname,
+                        usersInputAddress.housenumber
+                      )}`
```

Comment:
I'm noticing there are a bunch of places where you're essentially passing as arguments to `createRouteForAddressPage` an object that contains `boro`, `streetname`, and `housenumber` properties, but you have to split them up into sequential arguments.  I can think of a few ways to make the calls to this code a bit more concise:

* Change `createRouteForAddressPage` to take an object containing these properties, rather than sequential arguments.  This has the additional benefit of reducing the possibility for confusion of argument order: for example, if a developer mixed up the order and passed in `boro, housenumber, streetname` that would be bad, but it's much harder to make that mistake with named arguments.  The downside of this approach, of course, is that calls become a bit more verbose when you don't already have an object matching this schema on-hand.

* Overload `createRouteForAddressPage` to take _either_ sequential arguments _or_ an object with these properties.  This would require some [TypeScript overloading](https://www.typescriptlang.org/docs/handbook/functions.html#overloads) which gets a bit messy, though, and ideally you'd probably want to add some unit tests just to make sure the function behaves as you expect.

* Create a separate `createRouteForAddressPageFromObject()` function that takes the object with these properties.

Of course, you can also do nothing and keep these calls as they are, which is also totally fine!

Reply:
I ended up going with option one, which is a lot cleaner :) thanks